cobalt/media/media_module.cc

```diff
@@ -198,6 +198,16 @@ bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
     LOG(INFO) << (value ? "Enabling" : "Disabling")
               << " media metrics collection.";
     return true;
+  } else if (name == "EnableVideoBufferBudgetOverride") {
+    is_video_buffer_budget_override_enabled_ = value;
+    LOG(INFO) << (value ? "Enabling" : "Disabling")
```

Comment:
Consider allowing the web app to set the budget directly (i.e. 300MB, or 120MB) instead of setting a flag to use the maximum value.  This would give them more flexibility.

Reply:
Done, can be now be set to any value. In the SourceBufferStream constructor, it will take the std::max value between the override and the default value.